internal/server/compile.go

```diff
@@ -1418,6 +1422,10 @@ func (s *Server) inspectSpxSoundResourceRefAtExpr(result *compileResult, expr go
 		if _, ok := result.spxSoundResourceAutoBindings[obj]; !ok {
 			return nil
 		}
+		defIdent := result.defIdentFor(obj)
+		if defIdent == ident {
```

Comment:
It seems a normal business logic? need a test case to cover it.

Reply:
done https://github.com/goplus/goxlsw/compare/d5f978118ab22292526dc0f89b0f2b49503cfd69..4358dcc584a5ccfffb3ff4a13ad981c7c90e65f6